crates/uv-dispatch/src/lib.rs

```diff
@@ -219,6 +223,10 @@ impl BuildContext for BuildDispatch<'_> {
         &self.workspace_cache
     }
 
+    fn extra_build_dependencies(&self) -> &uv_workspace::pyproject::ExtraBuildDependencies {
```

Comment:
Sort of prefer using just `ExtraBuildDependencies` here rather than the fully-qualified path, for consistency.